src/responder/common/responder_iface.c

```diff
@@ -99,35 +99,20 @@ sss_resp_register_sbus_iface(struct sbus_connection *conn,
 {
     errno_t ret;
 
-    SBUS_INTERFACE(iface_resp_domain,
-        sssd_Responder_Domain,
-        SBUS_METHODS(
-            SBUS_SYNC(METHOD, sssd_Responder_Domain, SetActive, sss_resp_domain_active, rctx),
-            SBUS_SYNC(METHOD, sssd_Responder_Domain, SetInconsistent, sss_resp_domain_inconsistent, rctx)
-        ),
-        SBUS_SIGNALS(SBUS_NO_SIGNALS),
-        SBUS_PROPERTIES(SBUS_NO_PROPERTIES)
+    struct sbus_listener listeners[] = SBUS_LISTENERS(
+        SBUS_LISTEN_SYNC(sssd_Responder_Domain, SetActive,
+                         SSS_BUS_PATH, sss_resp_domain_active, rctx),
```

Comment:
Shouldn't signal handlers return 'void'?

Reply:
In theory, it can. But current code logs the result.

So no reply is send to the sender (because it is signal), but we still log if the handler succeeded or not. E.g.:

```
(2023-10-30 12:24:19): [sssd] [sbus_signal_handler] (0x2000): Received D-Bus signal org.freedesktop.DBus.NameOwnerChanged on /org/freedesktop/DBus
(2023-10-30 12:24:19): [sssd] [sbus_name_owner_changed] (0x4000): Name of owner sssd.sudo has changed from [] to [sssd.sudo]
(2023-10-30 12:24:19): [sssd] [sbus_issue_request_done] (0x0400): org.freedesktop.DBus.NameOwnerChanged: Success

```